### PR TITLE
Fixed S3 credentials access bug

### PR DESCRIPTION
**Pull request by dipanjanS**
Fixed S3 credentials access bug 632c781

### CODE REVIEW

File: rudra/data_store/bigquery/maven_bigquery.py [hunk 1]
```diff
@@ -41,7 +41,9 @@ def __init__(self, big_query_instance=None, s3_client=None,
         self.big_query_instance = big_query_instance or MavenBigQuery()
         self.big_query_content = list()
         self.counter = Counter()
-        self.bucket_name = 'developer-analytics-audit-report'
+        self.bucket_name = self.s3_client.bucket_name\
```
Comment (miteshvp) [inline rudra/data_store/bigquery/maven_bigquery.py]: self.bucket_name = self.s3_client.bucket_name or 'developer-analytics-audit-report'

File: rudra/data_store/bigquery/npm_bigquery.py [hunk 2]
```diff
@@ -36,7 +36,9 @@ def __init__(self, big_query_instance=None, s3_client=None,
         self.big_query_instance = big_query_instance or NpmBigQuery()
         self.big_query_content = list()
         self.counter = Counter()
-        self.bucket_name = 'developer-analytics-audit-report'
+        self.bucket_name = self.s3_client.bucket_name\
```
Comment (miteshvp) [inline rudra/data_store/bigquery/npm_bigquery.py]: same as above

File: rudra/data_store/bigquery/pypi_bigquery.py [hunk 3]
```diff
@@ -44,7 +44,9 @@ def __init__(self, big_query_instance=None, s3_client=None,
         self.big_query_instance = big_query_instance or PyPiBigQuery()
         self.big_query_content = list()
         self.counter = Counter()
-        self.bucket_name = 'developer-analytics-audit-report'
+        self.bucket_name = self.s3_client.bucket_name\
```
Comment (miteshvp) [inline rudra/data_store/bigquery/pypi_bigquery.py]: ditto again

File: rudra/data_store/bigquery/maven_bigquery.py (hunk 1 above)
Comment (dipanjanS) [reply rudra/data_store/bigquery/maven_bigquery.py]: oops that is a better way indeed, thanks will do!

**Comment by dipanjanS** [comment]
> Just some minor nitpick, if you want to change it. Else LGTM. Feel free to merge.

Thanks @miteshvp done! merging it now.

**Pull request merged**